### DISCUSSION TITLE
Force through parsing errors like IDEs?

### DISCUSSION THREAD

**Discussion by arash2060** [eslint/eslint#16778] [Q&A]
In VS Code or PyCharm, I get lint errors on undefined names and other issues despite e.g. having non-matching brackets, missing semi-colons, etc. However, `eslint_d` only throws "parsing error" at any of these issues. Is there a configuration to match the behavior of the IDE, or there's some other secret sauce to pre-process the file for eslint?

**Comment by sam3k** [comment]
Hi @arash2060, not sure it is clear whether the problem is with Eslint or `eslint_d`. If you think the issue is with Eslint, could you create a reproducible example _without `eslint_d`_ and put on a repo along with VSCode settings? If it does not reproduce without `eslint_d` then it is likely a problem with that package which we do not maintain.
